src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2119,8 +2119,17 @@ public sealed partial class String : System.Collections.Generic.IEnumerable<char
         public System.String ToUpper(System.Globalization.CultureInfo culture) { throw null; }
         public string ToUpperInvariant() { throw null; }
         public string Trim() { throw null; }
+#if netcoreapp11
```

Comment:
I thought I saw that explanation in another PR as well, but there were other netcoreapp11 ifdefs in this file, so I just matched that here.